lib/dust.js

```diff
@@ -270,18 +308,19 @@
     }
   };
 
-  function Context(stack, global, blocks, templateName) {
+  function Context(stack, global, options, blocks, templateName) {
     if(stack !== undefined && !(stack instanceof Stack)) {
       stack = new Stack(stack);
     }
     this.stack = stack;
     this.global = global;
+    this.options = options;
```

Comment:
not sure if it makes sense to default to `{}` here so that `onload` developers won't ever have to worry about an undefined object. It looks like this is handled already in the `makeBase` and `wrap` methods and it's consistent with how we are handling stack so feel free to ignore this.